### PR TITLE
#180 Added 'View comments' permission

### PR DESCRIPTION
**Pull request by razvanMiu**
Refs #180

### CODE REVIEW

**Review by avoinea — APPROVED** [review]

File: plone/app/discussion/profiles/default/rolemap.xml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0"?>
 <rolemap>
     <permissions>
+        <permission name="View comments" acquire="True">
```
Comment (fredvd) [inline plone/app/discussion/profiles/default/rolemap.xml:4]: @avoinea  You can add the permission to the rolemap.xml so the previous behavior  (comments visible for anonymous) is added again, but you would also need an upgrade step to apply this rolemap in existing sites when you upgrade. 

Not sure where is has to go, but I assume plone.app.upgrade.

Comment (avoinea) [reply plone/app/discussion/profiles/default/rolemap.xml:4]: @fredvd I see there is an `upgrade.zcml` within this package. Thus, I think it should go first here, @razvanMiu ^

Comment (avoinea) [reply plone/app/discussion/profiles/default/rolemap.xml:4]: @fredvd I see @razvanMiu already added the upgrade step here. 

I don't know about plone.app.upgrades, when and how it should go there :thinking: @mauritsvanrees @jensens @ale-rt

**Comment by razvanMiu** [comment]
@jenkins-plone-org please run jobs

File: plone/app/discussion/profiles/default/rolemap.xml (hunk 1 above)
Comment (jensens) [reply plone/app/discussion/profiles/default/rolemap.xml:4]: `plone.app.upgrades` can have a step loading the step from here.

Comment (mauritsvanrees) [reply plone/app/discussion/profiles/default/rolemap.xml:4]: `plone.app.discussion:default` is in the `ADDON_LIST` in `CMFPlone.MigrationTool`, so an upgrade is automatically applied during `@@plone-upgrade`.
I just tested the upgrade step here. Indeed needed. Before running it, anonymous cannot see comments. Afterwards, you can.

**Review by mauritsvanrees — APPROVED** [review]

**Pull request merged**